src/components/NcTextArea/NcTextArea.vue

```diff
@@ -71,7 +71,6 @@ It extends and styles an HTMLTextAreaElement.
 				:id="computedId"
 				ref="input"
 				class="textarea__input"
-				:type="type"
```

Comment:
Unrelated to PR

Reply:
Know, just was clearing up all the Vue warnings while adding docs